src/containers/Main.js

```diff
@@ -176,6 +176,7 @@ export default class Main extends Component {
 							)}
 						/>
 					</Switch>
+					<Top />
```

Comment:
You are displaying `<Top />` but you have not imported the component at the top. That's why build fails.